### ISSUE TITLE
package header info, and face suggestion for block groupings

### ISSUE THREAD

**Issue by GaryDahl** [jdtsmith/org-modern-indent#1]
Thanks for putting this nice package together.  It appears to resolve/work-around a [problem that I encountered with org-modern](https://github.com/minad/org-modern/issues/64)'s fringe bitmap display on mac.  While setting this up I noticed what looked like a couple of easy (possible) improvements that I wanted to share.

I noticed that quelpa will not install org-modern-indent the "--- package description" bit is missing from the middle of the first line in the .el package file.  I wasn't very familiar with this kind of requirement but found a short description here: https://www.gnu.org/software/emacs/manual/html_node/elisp/Library-Headers.html

The second thing that I noticed was that the brackets next to each block were rendering with small vertical spaces between each line.  After a bit of tinkering, I was able to fix this by changing the face that is being used to render these characters from 'org-meta-line to 'font-lock-comment-face.  It looks like 'org-meta-line was basically inheriting from both fixed-pitch and font-lock-comment-face, and that getting rid of the fixed-pitch helped remove these gaps between line drawing characters.  In case this is a font-specific problem, I'm using Jet Brains Mono.

**Comment by jdtsmith** [comment]
Thanks, added the description. 

For the font issue, that's what happens when line-spacing is non-zero; is that the case for you?  Can you `C-u C-x =` on the beginning of a block line and report back?  Also, `M-x describe-face org-meta-line [Ret]`.

**Comment by GaryDahl** [comment]
Great, quelpa is now able to install this package.

It looks like line-spacing is nil by default, and that changing this to value to zero does not change how these characters are displayed.  When I check the face description, it does not list the line-spacing.  Instead, this is what I see (and what motivated me to try changing the face):

```
Face: org-meta-line (sample) (customize this face)

Documentation:
Face for meta lines starting with "#+".

Defined in ‘org-faces.el’.


           Family: unspecified
          Foundry: unspecified
            Width: unspecified
           Height: unspecified
           Weight: unspecified
            Slant: unspecified
       Foreground: unspecified
DistantForeground: unspecified
       Background: unspecified
        Underline: unspecified
         Overline: unspecified
   Strike-through: unspecified
              Box: unspecified
          Inverse: unspecified
          Stipple: unspecified
             Font: unspecified
          Fontset: unspecified
           Extend: unspecified
          Inherit: (fixed-pitch font-lock-comment-face)

  This face was introduced, or its default value was changed, in
  version 22.1 of Emacs.
```

**Comment by jdtsmith** [comment]
Yeah nil=0. 

It's not clear why fixed-pitch is an issue, but I can confirm it.  Assigning fixed-pitch anywhere falls back to Monaco, with >1.0 line height:

<img width="38" alt="image" src="https://user-images.githubusercontent.com/93749/173213824-82f4e699-0083-43f4-85e2-56ec08a3efb0.png">

A good workaround is, e.g.:

```elisp
(set-face-attribute 'fixed-pitch nil :family "Hack" :height 1.0) ; or whatever font family
```

This will then work on org blocks too.   I hadn't noticed because my theme overrides org-meta-line.

**Comment by jdtsmith** [comment]
Closing this as fixed by e118f7e.
